Automattic/wp-calypso
# Add enhanced ownership settings

Related to https://github.com/Automattic/dotcom-forge/issues/3655 and https://github.com/Automattic/dotcom-forge/issues/3657

Based on https://github.com/Automattic/jetpack/pull/33081
## Proposed Changes

* Add enhanced ownership settings section when locked mode and legacy contact features are available.

### To do:
- [x] Make toggle and input functional.
- [x] Get copy review.

<img width="738" alt="Screen Shot 2023-09-12 at 3 26 05 PM" src="https://github.com/Automattic/wp-calypso/assets/1689238/1bb3beeb-a5ae-492d-a2ab-008725615a3a">


## Testing Instructions



* Sandbox public-api and a test site on Simple.
* Apply this Jetpack PR to your sandbox, using https://github.com/Automattic/jetpack/pull/33081#issuecomment-1719703029.
* Add a 100-year plan in your Store Admin to your test site.
* Go to /settings/general in Default mode.
* Check the Lock Mode option and Save Changes.
* Populate the Legacy Contact input field and Save Changes.
* Check the blog RC to make sure audit log entries appear as expected.
* Visit /wp-admin/options-general.php on your test site and scroll to the end of the page.
* The values you set in Calypso should show up there. They should also update when you change them from Calypso.


## Pre-merge Checklist
